Bug: `test_everflow_fwd_recircle_port_queue_check` fails specifically for `TestEverflowV4IngressAclIngressMirror` variant on VOQ systems

### Is it platform specific

generic

### Importance or Severity

High

### Description of the bug

`everflow/test_everflow_testbed.py::TestEverflowV4IngressAclIngressMirror::test_everflow_fwd_recircle_port_queue_check` fails due to unexpected number of packets in the Ethernet-Rec0 VOQ

```
            for q in range(1, 8):
                if str(q) == queue:
>                   assert wait_until(30, 1, 0, check_queue_counters, duthost, asic_ns, recircle_port, q, count), \
                        "Recircle port {} queue{} counter value is not same as packets sent".format(recircle_port, q)
E                   AssertionError: Recircle port Ethernet-Rec0 queue4 counter value is not same as packets sent
```

The test calls `send_and_check_mirror_packets` 10 times and therefor expects 10 packets to show up on Ethernet-Rec0's VOQ.
However we're seeing 20 packets:
```
# show queue counters Ethernet-Rec0
         Port    TxQ    Counter/pkts    Counter/bytes    Drop/pkts    Drop/bytes
-------------  -----  --------------  ---------------  -----------  ------------
Ethernet-Rec0    UC4              20            3,060            0             0
```

The reason being is on VOQ systems specifically for Ingress Everflow every call to `send_and_check_mirror_packets` actually injects 2 packets.
Seen here:
https://github.com/sonic-net/sonic-mgmt/blob/202511/tests/everflow/everflow_test_utilities.py#L1046
```
        if 't2' in setup['topo'] and 'lt2' not in setup['topo'] and 'ft2' not in setup['topo']:
            if valid_across_namespace is True:
                src_port_set.add(src_port) <-----------------------------------------
                src_port_metadata_map[src_port] = (None, 1)
                # Add the dest_port to src_port_set only in non MACSEC testbed scenarios
                if not MACSEC_INFO:
                    if duthost.facts['switch_type'] == "voq":
                        if self.mirror_type() != "egress":  # no egress route on the other node/namespace
                            src_port_set.add(dest_ports[0]) <-------------------------------------------------
                            src_port_metadata_map[dest_ports[0]] = (setup[direction]["egress_router_mac"], 1)
```
We add one of the `dest_ports` to the `src_port_set` meaning we'll have 2 entries in there resulting in 2 injected packets and 2 packets hitting Ethernet-Rec0.

### Steps to Reproduce

 Run `everflow/test_everflow_testbed.py::TestEverflowV4IngressAclIngressMirror::test_everflow_fwd_recircle_port_queue_check` on a VOQ system

### Actual Behavior and Expected Behavior

We should expect the Ethernet-Rec0 VOQ counter to be equal to the amount of packets we injected.

### Relevant log output

```shell

```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/everflow/everflow_test_utilities.py
+++ b/tests/everflow/everflow_test_utilities.py
@@ -1090,7 +1090,7 @@ class BaseEverflowTest(object):
 
                 if isinstance(result, bool):
                     logging.info("Using dummy testutils to skip traffic test, skip following checks")
-                    return
+                    return num_pkts_sent
 
                 _, received_packet = result
                 logging.info("Received packet: %s", packet.Ether(received_packet).summary())

